EXPO/screens/MapScreen.js

```diff
@@ -0,0 +1,447 @@
+        }
+      ]
+    },
+    {
+      featureType: "road",
+      elementType: "geometry",
+      stylers: [
+        {
+          color: "#f5f1e6"
+        }
+      ]
+    },
+    {
+      featureType: "road.arterial",
+      elementType: "geometry",
+      stylers: [
+        {
+          color: "#fdfcf8"
+        }
+      ]
+    },
+    {
+      featureType: "road.highway",
+      elementType: "geometry",
+      stylers: [
+        {
+          color: "#f8c967"
+        }
+      ]
+    },
+    {
+      featureType: "road.highway",
+      elementType: "geometry.stroke",
+      stylers: [
+        {
+          color: "#e9bc62"
+        }
+      ]
+    },
+    {
+      featureType: "road.highway.controlled_access",
+      elementType: "geometry",
+      stylers: [
+        {
+          color: "#e98d58"
+        }
+      ]
+    },
+    {
+      featureType: "road.highway.controlled_access",
+      elementType: "geometry.stroke",
+      stylers: [
+        {
+          color: "#db8555"
+        }
+      ]
+    },
+    {
+      featureType: "road.local",
+      elementType: "labels.text.fill",
+      stylers: [
+        {
+          color: "#806b63"
+        }
+      ]
+    },
+    {
+      featureType: "transit.line",
+      elementType: "geometry",
+      stylers: [
+        {
+          color: "#dfd2ae"
+        }
+      ]
+    },
+    {
+      featureType: "transit.line",
+      elementType: "labels.text.fill",
+      stylers: [
+        {
+          color: "#8f7d77"
+        }
+      ]
+    },
+    {
+      featureType: "transit.line",
+      elementType: "labels.text.stroke",
+      stylers: [
+        {
+          color: "#ebe3cd"
+        }
+      ]
+    },
+    {
+      featureType: "transit.station",
+      elementType: "geometry",
+      stylers: [
+        {
+          color: "#dfd2ae"
+        }
+      ]
+    },
+    {
+      featureType: "water",
+      elementType: "geometry.fill",
+      stylers: [
+        {
+          color: "#b9d3c2"
+        }
+      ]
+    },
+    {
+      featureType: "water",
+      elementType: "labels.text.fill",
+      stylers: [
+        {
+          color: "#92998d"
+        }
+      ]
+    }
+  ];
+  componentDidMount() {
+    Geocoder.init("AIzaSyD7JZmztK5wE-80P8t-_IOHZQinVtx4Dio");
+  }
+  // onPressLearnMore() {
+  // //  console.log(this.state.text);
+  // }
+
+  _getLattLong(locationName) {
+    console.log("getLatsstLongs called "+locationName);
+    Geocoder.from(locationName)
+          .then(json => {
+              var location = json.results[0].geometry.location;
+              console.log("Location :::"+location);
+              this.pinPointSearchedLocation(location);
+          })
+          .catch(error => console.warn(error));
+  }
+
+  pinPointSearchedLocation(location) {
+    console.log("Pinporint location"+location);
+   this.setState({
+    latLng: {
+      latitude: location.latitude,
+      longitude: location.longitude
+    }
+    });
+  }
+
+  componentWillMount() {
```

Comment:
use componentDidMount, we can not do setState before component mount

Reply:
Changes has been done as per suggestions.